.github/CONTRIBUTING.md

```diff
@@ -52,6 +52,7 @@ Then clone the sources and install node modules:
 
 ```
 $ git clone https://github.com/Chocobozzz/PeerTube
+$ git checkout develop
```

Comment:
You don't need to checkout develop, if what you are testing is in an other branch you checkout the other branch.

Reply:
Right, I'm adding a comment regarding this.